distribution/ddf-common/src/main/resources/etc/users.attributes

```diff
@@ -1,8 +1,8 @@
 {
     "admin" : {
-
+        "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress":"admin@localhost.local"
     },
     "localhost" : {
-
+        "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress":"localhost@localhost.local"
```

Comment:
The host names in the email addresses should be updated by the installer.  @stustison recently fixed the key value being updated in the installer.

https://github.com/codice/ddf/pull/609